### ISSUE TITLE
ImportOrderCheck false positive

### ISSUE THREAD

**Issue by cesares-basilico** [checkstyle/sonar-checkstyle#123]
Sonar version: 6.4
Checkstyle sonar plugin verson: 3.7

Steps to reproduce or description of problem:

I frequently get some false positive by ImportOrderCheck like that one:

```
import it.xxx.constants.ConfigurationParametersKeys;
import it.xxx.enumeration.BiometricsResponses;
```

this one seems to respect the rules, package alphabetical order and then class alphabetical order

**Comment by romani** [comment]
All validation is done by checkstyle library , please open issue at https://github.com/checkstyle/checkstyle/issues pay attention to required format of issue report- it helps a lot
